### DISCUSSION TITLE
Obsolete scenarios and recommended roadmap

### DISCUSSION THREAD

**Discussion by LevT2** [Microsoft/WSLab#528] [Q&A]
Could you provide a roadmap through the scenarios? 

For example, I noticed some of them build on the "S2D Hyperconverged" while itself refers to the "AzSHCI Deployment". Yet we need to discard the former VMs, to rebuild the VMs with the correct LabConfig.

**Comment by LevT2** [comment]
Apparently, I was requesting these updated guides: https://github.com/DellGEOS/AzureStackHOLs

**Reply by jaromirk** [reply]
Yep. I keep scenarios here and I more or less maintain as HOLs. All should works, some are less relevant, but still works.

**Reply by LevT2** [reply]
It is a lot of work and required knowledge to adapt an obsolete scenario, e.g. to trace the right parent disks in the LabConfigs. I want them upgraded, being not sure the lab would behave as it was expected years ago.
